README.md

```diff
@@ -78,15 +78,25 @@ XAMPP is a very easy to install Apache Distribution for Linux, Solaris, Windows
 
 This [video](https://youtu.be/Yzksa_WjnY0) walks you through the installation process for Windows but it should be similar for other OSs.
 
-### Config File
+### Docker
 
-DVWA ships with a dummy copy of its config file which you will need to copy into place and then make the appropriate changes. On Linux, assuming you are in the DVWA directory, this can be done as follows:
+It is possible to run DVWA with containers.
 
-`cp config/config.inc.php.dist config/config.inc.php`
+Prerequisites: Docker and Docker Compose.
 
-On Windows, this can be a bit harder if you are hiding file extensions, if you are unsure about this, this blog post explains more about it:
+- If you are using Docker Desktop, both of these should be already installed.
+- If you prefer Docker Engine on Linux, make sure to follow their [installation guide](https://docs.docker.com/engine/install/#server).
```

Comment:
You need to add something else to point out that you are directing them at this version as it may not be the one that comes with the system package manager.

I was going to suggest telling them how to uninstall that version, but what happens if they have containers already setup using the distro package version? Will installing this different version break all of those?

Reply:
Containers are just processes, and uninstalling Docker only means losing the tool to manage the containers. Still, Docker has backwards compability, so containers shouldn't just break.